docs/src/model_creation/examples/basic_CRN_library.md

```diff
@@ -1,4 +1,22 @@
 # [Library of Basic Chemical Reaction Network Models](@id basic_CRN_library)
+```@raw html
+<details><summary><strong>Tutorial environment set-up</strong></summary>
```

Comment:
Is technically everything at tutorial though? Maybe we write "Environment set-up" and keep it at that, or something else. Or "Julia environment set-up".

Reply:
Maybe say "Environment setup and package installation"?

Reply:
That sound good